static/js/actions/email.js

```diff
@@ -31,6 +31,23 @@ export const sendEmailSuccess = createAction(SEND_EMAIL_SUCCESS);
 export const SEND_EMAIL_FAILURE = 'SEND_EMAIL_FAILURE';
 export const sendEmailFailure = createAction(SEND_EMAIL_FAILURE);
 
+export function sendEmail(
+  emailType: string,
```

Comment:
it can theoretically be any string. `static/js/components/email/constants.js` has a short list of strings that we can use for known email types, but this argument to `sendEmail` just depends on the keys passed into the `withEmailDialog` higher-order component (see: `static/js/containers/DashboardPage.js:732`). those keys can be anything